pkg/build/controller/config_controller.go

```diff
@@ -28,8 +28,12 @@ func (e ConfigControllerFatalError) Error() string {
 
 // IsFatal returns true if err is a fatal error
 func IsFatal(err error) bool {
-	_, isFatal := err.(ConfigControllerFatalError)
-	return isFatal
+	switch err.(type) {
+	case ConfigControllerFatalError, *ConfigControllerFatalError:
```

Comment:
I think this is bad news - is it possible to fix the one place where we return &ConfigControllerFatalError (as opposed to ConfigControllerFatalError) instead?